### PR TITLE
Updates to latest libraries and adjusts for deprecation

### PR DESCRIPTION
**Pull request by adriancole**

### CODE REVIEW

File: module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java [hunk 1]
```diff
@@ -51,8 +47,6 @@
 
   @Rule public MockitoRule mocks = MockitoJUnit.rule();
```
Comment (adriancole) [inline module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java]: later commit kills this rule

File: brave-instrumentation/aws-java-sdk-core/src/main/java/brave/instrumentation/aws/AwsClientTracing.java [hunk 2]
```diff
@@ -98,4 +103,65 @@ public static AwsClientTracing create(HttpTracing httpTracing) {
...
+    @Override public String method() {
+      return delegate.getHttpMethod().name();
+    }
+
+    @Override public String path() {
+      return delegate.getResourcePath();
+    }
+
+    @Override public String url() {
+      StringBuilder url = new StringBuilder(delegate.getEndpoint().toString());
```
Comment (adriancole) [inline brave-instrumentation/aws-java-sdk-core/src/main/java/brave/instrumentation/aws/AwsClientTracing.java]: all this code lifted directly from the previous adapter code

File: brave-instrumentation/aws-java-sdk-v2-core/src/main/java/brave/instrumentation/awsv2/AwsSdkTracing.java [hunk 3]
```diff
@@ -31,4 +36,76 @@ public static AwsSdkTracing create(HttpTracing httpTracing) {
...
+    @Override public String method() {
+      return delegate.method().name();
+    }
+
+    @Override public String path() {
+      return delegate.encodedPath();
+    }
+
+    @Override public String url() {
+      StringBuilder url = new StringBuilder(delegate.protocol())
```
Comment (adriancole) [inline brave-instrumentation/aws-java-sdk-v2-core/src/main/java/brave/instrumentation/awsv2/AwsSdkTracing.java]: all this code lifted directly from the previous adapter code

File: module/collector-sqs/src/test/java/zipkin/module/collector/sqs/ZipkinSQSCollectorModuleTest.java [hunk 4]
```diff
@@ -58,7 +55,7 @@ public void close() {
     if (context != null) context.close();
   }
 
-  @Test
+  @Test(expected = NoSuchBeanDefinitionException.class)
```
Comment (anuraaga) [inline module/collector-sqs/src/test/java/zipkin/module/collector/sqs/ZipkinSQSCollectorModuleTest.java]: I guess we'd usually use `assertThatThrownBy`

File: module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java [hunk 5]
```diff
@@ -106,13 +95,14 @@ void expectSignatureInterceptor() {
...
-
-    // TODO(anuraaga): Can be simpler after https://github.com/line/armeria/issues/1883
+    ClientOptionsBuilder clientBuilder = ClientOptions.builder();
     customizer.accept(clientBuilder);
-    Client<HttpRequest, HttpResponse> decorated = clientBuilder.build().decoration()
-        .decorate(HttpRequest.class, HttpResponse.class, mockHttpClient);
-    assertThat(decorated).isInstanceOf(AWSSignatureVersion4.class);
+
+    WebClient client = WebClient.builder("http://127.0.0.1:1234")
+        .option(ClientOption.DECORATION, clientBuilder.build().decoration())
```
Comment (anuraaga) [inline module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java]: Think it's more precise to do `.options(clientBuilder.build())`

File: module/storage-xray/src/test/java/zipkin/module/storage/xray/ZipkinXRayStorageModuleTest.java [hunk 6]
```diff
@@ -38,15 +34,14 @@ public void close() {
     }
   }
 
-  @Test
+  @Test(expected = NoSuchBeanDefinitionException.class)
```
Comment (anuraaga) [inline module/storage-xray/src/test/java/zipkin/module/storage/xray/ZipkinXRayStorageModuleTest.java]: Ditto

File: module/collector-sqs/src/test/java/zipkin/module/collector/sqs/ZipkinSQSCollectorModuleTest.java (hunk 4 above)
Comment (adriancole) [reply module/collector-sqs/src/test/java/zipkin/module/collector/sqs/ZipkinSQSCollectorModuleTest.java]: imho overkill when only testing the exception class..

File: module/storage-xray/src/test/java/zipkin/module/storage/xray/ZipkinXRayStorageModuleTest.java (hunk 6 above)
Comment (adriancole) [reply module/storage-xray/src/test/java/zipkin/module/storage/xray/ZipkinXRayStorageModuleTest.java]: ditto also :P

File: module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java (hunk 5 above)
Comment (adriancole) [reply module/storage-elasticsearch-aws/src/test/java/zipkin/module/storage/elasticsearch/aws/ZipkinElasticsearchAwsStorageModuleTest.java]: excellent I like less lines and imports for the same job

**Comment by adriancole** [comment]
thx for the help @anuraaga

**Pull request merged**